src/components/tiles/TilesList.tsx

```diff
@@ -42,30 +50,30 @@ export const TilesList = ({
 
   return (
     <ScrollView testID={tID('tiles-list')} style={styles.scrollView}>
-      <View style={styles.view}>
-        {tiles?.map((appTile) => (
-          <Tile
-            key={appTile.id}
-            id={appTile.id}
-            title={appTile.title}
-            onPress={
-              onAppTilePress ? onAppTilePress : onAppTilePressDefault(appTile)
-            }
-            styles={tileStyles}
-          />
-        ))}
-        {children}
-      </View>
+      {tiles?.map((appTile) => (
+        <Tile
+          id={appTile.id}
+          key={appTile.id}
+          title={appTile.title}
+          onPress={
+            onAppTilePress ? onAppTilePress : onAppTilePressDefault(appTile)
+          }
+          Icon={appTileIcon(appTile.icon)}
```

Comment:
Is `appTile.icon` guaranteed to be an svg? What happens if it is a jpg/png?

Reply:
I'll double check.

Reply:
✅